### DISCUSSION TITLE
Question regarding the possibility

### DISCUSSION THREAD

**Discussion by DavorZg** [shridhar-tl/jira-assistant#240] [Q&A]
Hi,

Do you have a possibility in the tool to provide info on when the user logged the time on ticket.
So not only on which date, but also the view when did he performed this activity. 

Thank you,
Davor

**Comment by shridhar-tl** [comment]
As of now it is not available. But that should be possible. You can provide some suggestions on what would be the appropriate place in worklog report to provide these details.

Providing it in flat worklog tab of worklog report would be the most easiest option. Additionally, if you just wanted to know the log time of individual worklogs, it would be very simple to get it implemented using custom report or report builder using expressions.

Let me know what/where exactly you are expecting for this information and I can help you with getting it.

**Comment by DavorZg** [comment]
Great news :).

What ever is easier for you, but I would appreciate to have also the option to filter for example worklogs that are **entered** in particular month, currently i have te possibility to see worklog that is for certain day/week/month.

I hope you understand :), because when someone is monitoring the project and looing at budget side 
it is important to know if someone entered the worklog after the "freeze time".

**Comment by shridhar-tl** [comment]
Hi @DavorZg - just to understand bit more about it, do you want to just ensure that you see worklogs created between a specific date range or you want to see worklogs on daily basis based on created date?

I mean, for example worklog is created on 12th Nov for 30 Sep. Currently that worklog is shown under 30Sep. Do you want this to be shown under 12th Nov instead or do you expect to just filter out this worklog and do not show it based on some date range?

**Reply by DavorZg** [reply]
My main goal is to be able to see when someone enters the worklog after the certain date. 
So if we for example finish/end the October, I would like to be able to see if someone entered the worklog in November for October.

Thank you

**Reply by shridhar-tl** [reply]
I'm currently working on a new worklog report and would incorporate this enhancement as well along with it.

**Reply by DavorZg** [reply]
Great, please let me know when you make the changes

**Reply by shridhar-tl** [reply]
Hi @DavorZg, I have completed the implementation. By today or by tomorrow, it would be first released in Web version of JA where you should be able to start using it just by accessing https://app.jiraassistant.com.

After few days, once I feel changes are stable enough, I would publish it for extension users.

Here is the screenshot of the Settings page where you should be able to configure how you can filter the data based on created / updated date.
![image](https://user-images.githubusercontent.com/37339683/205218126-81daa9f5-abf7-454d-8fb3-65f5d8fec1d2.png)

This is available only in new (BETA) worklog report and not in old worklog report. Also remember this is totally new report along with this implementation and you must verify if report is working as expected before using the data for any other purpose. I will do more testing by this weekend and fix any issues which I come across.

Please feel free to report any issue / suggestion you come across.

**Comment by DavorZg** [comment]
Hi,

Whoooo hooo, great.
Please let me know when I will be able to test.

Kind regards,
Davor

**Reply by shridhar-tl** [reply]
Hi @DavorZg, changes are already available in web version of JA. If you want to test/use the latest changes, you can use the web version by access https://app.jiraassistant.com.

**Comment by DavorZg** [comment]
Hi,

Can you please let me know where do I need to click:

[image: image.png]

When I click on "Worklog Report beta" blank screen is presented.

Kind regards,
Davor

**Reply by shridhar-tl** [reply]
Hi @DavorZg, I was not able to see the image you have posted, however I believe I understood what you mean. There was a minor issue for first time user which I resolved late evening yesterday, but just realized I haven't published it.
I just published those changes and hopefully it should get resolved.

If you still face this issue, please send me the screenshot of console error if any along with the issue you see on screen.

**Comment by DavorZg** [comment]
Hi,

Great, I think it works :). Thank you
Is there a way to see also when someone entered the worklog data?

For example to be able to see, that today someone entered 5h on ticket for
date 13.11.

Kind regards,
Davor

**Reply by shridhar-tl** [reply]
Hi @DavorZg - I need to think through it. One possibility is I can show created date also in Flat worklog tab where you can see created date of individual worklog.

If you have any suggestion you can let me know and I will see if that is feasible.

**Comment by shridhar-tl** [comment]
I hope this issue has been resolved long back and hence closing this ticket. Please feel free to reopen it if you have any other queries.
